bedrock/firefox/redirects.py

```diff
@@ -529,12 +529,12 @@ def firefox_channel(*args, **kwargs):
     # issue 8641
     redirect(r"^/firefox/windows-64-bit/?$", "firefox.browsers.windows-64-bit"),
     redirect(r"^/firefox/best-browser/?$", "firefox.browsers.best-browser"),
-    # Unfck campaign
-    redirect(r"^firefox/unfuck/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/love/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/liebe/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/rendonslenetplusnet/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^(unfu?ck|love|liebe|rendonslenetplusnet)/?$", "firefox.campaign.unfck.index"),
+    # Unfck campaign, issue 11613
+    redirect(r"^firefox/unfu?ck/?$", "firefox"),
```

Comment:
**Question:** why is there a `?` in the middle of `unfck`? Is it in the case if it was spelled with a `u` (making it the correct spelling 😝) to still redirect?

Reply:
It's a regular expression., so `?` means the preceding character is optional (meaning it matches both `/unfuck/` and `/unfck/`